odkx-src/trouble-shooting.rst

```diff
@@ -0,0 +1,38 @@
+.. spelling::
+    
+  Tecno
+  Infinix
+
+
+Troubleshooting
+=====================
+
+In this guide, we cover the likely problems you *might* encounter installing and configuring the ODK-X Basic Tools. The ODK-X Basic Tools are ODK-X Services, ODK-X Tables and ODK-X Survey. 
+
+Do ensure that you have followed all the instructions for installation stated in the :doc:`basics-install` page.
+
+.. _troubleshoot-devices:
+
+Devices
+--------------------
+
+This section covers issues that are specific to the Android device you are working with and how to solve these problems.
+
+.. note::
+  The ODK-X Basic Tool require an Android device with a version 4.4 or higher operating system. 
+    
+Certain **Tecno** and **Infinix** devices run specific system settings which prevents applications from running in the background to conserve battery life and prevent the device from lagging, especially on low memory devices.
+
+You will typically run into issues as a result of these devices system setting.

```

Comment:
devices' (apostrophe for possessive)

Reply:
Also here or in the previous setting you may want to describe how the problem appears to the user, something like "If you are using a Tecno or Infinix phone and are stuck on 'configuring' when opening the ODK-X tools" Then if users search for "configuring" (which is what they see on the tab) in the docs they are more likely to get this.